run-dljc.sh

```diff
@@ -6,14 +6,16 @@
 ### Usage
 
 # - Move this script to an experiment directory.
-# - Make a file containing a list of git repositories, one per line. Repositories must be of the form: https://github.com/username/repository - the script is reliant on the number of slashes, so excluding https:// is an error.
+# - Make a file containing a list of git repositories, one per line.
+#   Repositories must be of the form: https://github.com/username/repository
+#   The script is reliant on the number of slashes, so excluding https:// is an error.
 # - Ensure that your JAVA8_HOME variable points to a Java 8 JDK
 # - Ensure that your JAVA11_HOME variable points to a Java 11 JDK
 # - Ensure that your JAVA_HOME variable points to the same place as JAVA11_HOME
 # - Ensure that your CHECKERFRAMEWORK variable points to a built copy of the Checker Framework
 # - Other dependencies: perl, python2.7, awk, git, mvn
```

Comment:
`gradle` is another dependency, once you change the code to try using `gradle` as well as `./gradlew`.